lib/parity/environment.rb

```diff
@@ -37,7 +41,9 @@ def backup
     end
 
     def deploy
-      if deploy_to_heroku && run_migration?
+      pending_migrations = run_migration?
+
+      if deploy_to_heroku && pending_migrations
         migrate
       end
```

Comment:
I *think* the problem here is that if there are no pending migrations, this will return `nil`, because that's what `if` returns.

I think the behavior we want for this is:

```ruby
def deploy
  deploy_to_heroku && migrate_if_needed
end

def migrate_if_needed
  if run_migration?
    migrate
  else
    true
  end
end
```